modin/experimental/core/storage_formats/hdk/query_compiler.py

```diff
@@ -857,9 +857,9 @@ def dtypes(self):
         return self._modin_frame.dtypes
 
 
-_SUPPORTED_NUM_TYPE_CODES = set(np.typecodes["AllInteger"] + np.typecodes["Float"]) - {
-    np.dtype(np.float16).char
-}
+_SUPPORTED_NUM_TYPE_CODES = set(
+    np.typecodes["AllInteger"] + np.typecodes["Float"] + "?"
```

Comment:
Let's give name for `"?"` typecode.

Reply:
I didn't find a dedicated constant for the boolean typecode. I can add an inline comment here. Is it ok?

Reply:
Yes